### DISCUSSION TITLE
Erro API V2

### DISCUSSION THREAD

**Discussion by alanjmm** [BrasilAPI/BrasilAPI#608] [General]
Olá de uns dias pra cá ao tentar acessar a api V2  (Exp: https://brasilapi.com.br/api/cep/v2/87065460 ) recebo esse erro: 

{"message":"invalid json response body at https://nominatim.openstreetmap.org/search?format=json&addressdetails=1&country=Brasil&state=PR&city=Maring%C3%A1&street=Rua%20Piapara reason: Unexpected token < in JSON at position 0","type":"invalid-json"}

não importa qual cep eu busque, tem mais alguem recebendo esse erro?

**Comment by Stake2** [comment]
Se você acessar a página esta página diretamente:
https://nominatim.openstreetmap.org/search?format=json&addressdetails=1&country=Brasil&state=PR&city=Maring%C3%A1&street=Rua%20Piapara

Você vai receber as informações do endereço corretamente
Mas se acessar pela BrasilAPI na rota "cep", vai receber o erro que você citou
Acho que é a forma com que a rota "cep" passa os dados de request (o cep) para o "openstreetmap.org" que está errada
Ou a forma como a rota "cep" trata a resposta que ela recebe do "openstreetmap.org", já que a resposta vem em JSON
Então a resposta em JSON pode estar sendo mal interpretada pela rota "cep"

**Comment by vito-ysl** [comment]
Sim, mais gente está ou pode estar recebendo esse erro.
Não é problema seu — é problema no backend da BrasilAPI ou no Nominatim.

**Reply by vito-ysl** [reply]
Sim, mais gente está ou pode estar recebendo esse erro.
Não é problema seu — é problema no backend da BrasilAPI ou no Nominatim.
